### DISCUSSION TITLE
Welcome to InfraBoard Discussions!

### DISCUSSION THREAD

**Discussion by aryasoni98** [xerocodee/InfraBoard#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Avi2492** [comment]
I think the issue is in .env file as I read about this error on appwrite - The user encountered a TypeError: Failed to construct 'URL': Invalid URL issue. They received help and were able to fix the problem. The issue was caused by extra quotes in the .env file. The solution was to remove the extra quotes when setting the value.

**Comment by Vishnuteja10** [comment]
Hi iam vishnuteja, i want to contribute to this project, help me to get started
